includes/Dashboard.php

```diff
@@ -229,6 +229,9 @@ public function get_dashboard_settings() {
 			'config' => [
 				'isRTL'        => is_rtl(),
 				'dateFormat'   => get_option( 'date_format' ),
+				'timeFormat'   => get_option( 'time_format' ),
+				'gmtOffset'    => get_option( 'gmt_offset' ),
+				'timezone'     => get_option( 'timezone_string' ),
```

Comment:
Do we need both the offset and the timezone string?

There's a function called  `wp_timezone_string()` that we could use instead.

It uses the `timezone_string` option to get a proper timezone if available, otherwise falls back to the offset.

This way you don't have to do that check in JS.

Reply:
moment uses timezones and offsets differently.  We have `.utcOffset` for `gmt_offset` and `.tz` for `timezone`. We'd have to do a check anyways to see if whatever `wp_timezone_string` returns is a valid timezone or just always convert it to an offset to have consistency.